src/frontend/src/components/VerticalDetailDisplay.tsx

```diff
@@ -16,13 +16,34 @@ interface VerticalDetailDisplayProps {
 const VerticalDetailDisplay: React.FC<VerticalDetailDisplayProps> = ({ label, content, boxStyle }) => {
   const theme = useTheme();
   const backgroundColor = theme.palette.mode === 'dark' ? theme.palette.grey[800] : theme.palette.grey[200];
+
+  // Box Styles
+  const customSx = {
+    overflow: 'auto',
+    whiteSpace: 'nowrap',
+    backgroundColor: backgroundColor,
+    justifyContent: 'center',
+    boxShadow: 1,
+    borderRadius: '10px'
+  };
+
+  // Custom Scrollbar Styles
+  const hoverSx = {
```

Comment:
I think you should make the scrollbar style also a prop and implement it in a similar way

Reply:
i just now decided to include the scrollbar style (3 elements) in the sx instead of making it separate. hopefully this addresses your comments.